src/MspConf.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef MSP_CONF_HPP
+#define MSP_CONF_HPP
+
+////////////////////////////////////////////////////////////////////////////
+//
+// Configuration for MSP flight controller node.
+//
+////////////////////////////////////////////////////////////////////////////
+
+namespace FcCommsMspConf
```

Comment:
This should be in the `FcComms` namespace, we don't need separate namespaces for every implementation